packages/flutter/lib/src/widgets/text.dart

```diff
@@ -353,7 +364,7 @@ class Text extends StatelessWidget {
   /// will not be rendered. Otherwise, it will be shown with the given overflow option.
   const Text(
     this.data, {
```

Comment:
@a14n, could this be `String this.data` so that you can't pass nulls to the default constructor (i.e. `Text(null)` should be an error) ?

Reply:
Sure! Thanks for the review :) A lint could make sense to catch similar issue.

Reply:
Done. I also changed `Text.rich` textSpan parameter.

Reply:
Filed dart-lang/linter#2237